src/main_nep/parameters.cuh

```diff
@@ -84,6 +84,7 @@ public:
   std::vector<float> q_scaler_cpu;    // used to scale some descriptor components (CPU)
   std::vector<std::string> elements;  // atom symbols
   std::vector<int> atomic_numbers;    // atomic numbers
+  std::vector<float> parameters_cpu;  // GPU parameters, used for reporting in nep.txt (CPU)
```

Comment:
This `parameters_cpu` is not used. If the purpose is to report parameters to `nep.txt`, we don't need this quantity.